gtm/src/info.json

```diff
@@ -0,0 +1,14 @@
+    "displayName": "Algolia",
```

Comment:
Algolia Insights may be a more accurate name?

Reply:
This is the name of the brand, I think Algolia makes sense?

<details>
<summary>See tag types</summary>

![image](https://user-images.githubusercontent.com/6137112/63438957-81194480-c42d-11e9-85e8-6fca6cc66061.png)

</details>

Note: I'm gonna rename the template to "Search Insights" since this `displayName` already mentions Algolia.

Reply:
ok for the original point.

About the note: on your screen shot, the display name has the brand in it for a lot of them
![image](https://user-images.githubusercontent.com/2982512/63441855-5da4c880-c432-11e9-8d6b-876c6c667cf8.png)
![image](https://user-images.githubusercontent.com/2982512/63441899-7319f280-c432-11e9-91e2-8d25800cbd37.png)

Reply:
That's true!

02f1f2ebba80faf873f811169253f5e0fec7e5f3